xbmc/settings/AdvancedSettings.h

```diff
@@ -383,6 +383,8 @@ class CAdvancedSettings : public ISettingCallback, public ISettingsHandler
 
     std::string m_userAgent;
 
+    bool m_waitForNetAfterWakeUp;
```

Comment:
Consider initializing member variable when declaring it: `bool m_waitForNetAfterWakeUp = true;`